popup.js

```diff
@@ -111,28 +143,28 @@ function nominationTool( e ) {
   data[DATABASE_ID] = databaseID;
   data[COMMMENT_ID] = commentID;
 
-  // Do GET call to post to Google Form and open new tab
-  $.get( {
-    url: GOOGLE_FORMS_URL,
-    data,
-    success: function( res ) {
-      $( '#success' ).html( "Success!" );
+  $.get(GOOGLE_FORMS_URL, data)
+    .then(() => {
+      $('#save').prop('disabled', false);
+
+      showStatus('success', 'URL submitted. Thanks!');
+      // @todo Why are we reloading the popup here? To reset some fields?
```

Comment:
I'm not sure but I think part of the reason was to just flash the success field. But people often miss the notification, so actually maybe it's not good UI.